helmfile/charts/networkpolicy-common/templates/starboard/allow-starboard-operator.yaml

```
@@ -0,0 +1,25 @@
+{{ if .Values.monitoring.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-starboard-operator
+  namespace: monitoring
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/instance: starboard-operator
+      app.kubernetes.io/name: starboard-operator
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress: []
+  egress:
```

The files are a bit inconsistent if you are adding `ingress` or not when ingress should just be denied. I think the consensus is to not add it (as my suggestion below). So please check that all files handle this in the same way.
```suggestion
  policyTypes:
    - Egress
  egress:
```